content/docs/ops/secrets.md

```diff
@@ -66,21 +66,40 @@ All UAA clients and users and associated credentials should be created via the C
 
     ```bash
     mkdir -p tmp
-    aws s3 cp s3://cloud-gov-varz/cf.yml tmp/cf.yml.enc --profile govcloud
+    aws s3 cp s3://cloud-gov-varz/cf.main.yml tmp/cf.main.yml.enc --profile govcloud
     ```
 
 1. Get the encryption passphrase.
-    1. View the `deploy-cf` pipeline configuration.
+    1. View the `deploy-cf` pipeline configuration for any of the `common-*`
+       environments. The following command outputs all the `cg-common`
+       resources. You can modify the test for `test("common-")` to grab specific
+       credentials about each resource.
 
         ```bash
-        fly -t <target> get-pipeline -p deploy-cf | less
+        fly --target <target> get-pipeline --pipeline deploy-cf | \
+        spruce json | \
+        jq -r '.resources[] | select(.name | test("common-"))'
+        ```
+
+    1. Grab the `secrets_passphrase` under the **common-prod** resource. The
+       following command grabs just the **common-prod** resource and parses out
+       the `secrets_passphrase` found in the `pipeline.yml` file for
+       `cg-deploy-cf`. This is useful for feeding this value to other commands
+       without visually exposing the secret in your terminal.
+
+        ```bash
+        fly --target <target> get-pipeline --pipeline deploy-cf | \
+        spruce json | \
+        jq -r '.resources[] | select(.name | test("common-prod")) | .sources.secrets_passphrase'
         ```
 
-    1. Grab the `secrets_passphrase` under the `common-prod` section.
 1. Decrypt the secrets file.
 
     ```bash
-    INPUT_FILE=tmp/cf.yml.enc OUTPUT_FILE=tmp/cf.yml PASSPHRASE=... ./decrypt.sh
+    INPUT_FILE=tmp/cf.main.yml.enc
+    OUTPUT_FILE=tmp/cf.main.yml
+    PASSPHRASE=$(fly --target <target> get-pipeline --pipeline deploy-cf | spruce json | jq -r '.resources[] | select(.name | test("common-prod")) | .sources.secrets_passphrase')
```

Comment:
I think this would be a good thing to update after the fact. I only touched on this file a bit to address `main` / `external` files.